### DISCUSSION TITLE
Plans for the latest alpha and general state of project?

### DISCUSSION THREAD

**Discussion by TijuanaKez** [RomanKubiak/ctrlr#500] [Releases]
Testing the new alpha build, I really like the MIDIFX version for Logic.
But the problem is it can't open any MIDI devices (at least on Monterey)

I was able to build the latest source by adding JUCE_MODAL_LOOPS_PERMITTED=1 and specifying Plugin AU Main Type, but it has  other problems (Midi Device menu is empty).

Suspecting JUCE modules probably need updating, I tried with 7.0.2 it won't build due to Lua extending classes which are now marked 'final' (LFile).

EDIT: well I found /Scripts/remove_finals_from_JUCE.sh which fixed a lot. But those Lua extensions still cause too much havoc :(

I'm tempted to persist, but it's probably ultimately beyond may capabilities.

Could there be an update around the corner?

**Comment by damiensellier** [comment]
Hi,
Welcome to the CTRLR community. 
Unfortunately, there's no official timeline for the next release, we all try to fix bugs here and there from the current version and had them in the "Pull Requests" queue. 
Obviously CTRLR would require an update to the latest version of JUCE, Apple M1/M2 support and many other improvements.
I worked on the AAX version of the plugin but still wait for the different approvals from AVID because of the different kind of licencing involved. 
The creator of CTRLR Roman Kubiak left the project as-is and is MIA right now, so nothing major to be expected for short/medium term.

Don't hesitate to ask if you have any question.

Have a good one!

Damien
